src/main/java/domain/Lotto.java

```diff
@@ -0,0 +1,49 @@
+package domain;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class Lotto {
+    public static int PRICE = 1000;
+    private LottoNumbers lottoNumbers;
+
+    public Lotto(LottoNumbers lottoNumbers) {
+        this.lottoNumbers = lottoNumbers;
+    }
+
+    public int countMatch(LottoNumbers comparedNumbers) {
+        Set<LottoNumber> mergedLottoNumbers = mergeIntoSet(comparedNumbers, this.lottoNumbers);
+        return 2 * LottoNumbers.LOTTO_NUMBER_COUNT - mergedLottoNumbers.size();
+    }
```

Comment:
```suggestion
    public int countMatch(LottoNumbers comparedNumbers) {
        return this.lottoNumbers.match(comparedNumbers);
    }
```

- LottoNumbers 객체에 메시지를 던지는 것은 어떨까요